src/Neuralm.Services/Neuralm.Services.TrainingRoomService/Neuralm.Services.TrainingRoomService.Domain/TrainingRoom.cs

```diff
@@ -120,62 +120,57 @@ public void AddOrganism(Organism organism)
             Species.Add(new Species(organism, Id));
         }
 
+        // FIXME: Require Species to have an IsEvaluated property
```

Comment:
It still needs a mechanism to find if the species is evaluated using a property. This can be done in a later issue.